toolkit/tools/depsearch/depsearch.go

```diff
@@ -64,7 +64,7 @@ func main() {
 	}
 
 	if !(*maxDepth == -1 || *maxDepth >= 1) {
-		logger.Log.Fatalf("Invalid max depth '%d', valid ranges are -1, >=1", *maxDepth)
+		logger.Log.Fatalf("Invalid max depth (%d), valid ranges are -1, >=1", *maxDepth)
```

Comment:
For non-string variables, we skip drop both quotes and brackets - I think this one was added by mistake. They always have a value, so there's no risk of ending up with an empty string.

Reply:
Ok, thanks for letting me know.
I find () useful to know this is a variable's output, not a hardcoded value. Do you still want to drop it?